adafruit_displayio_sh1107.py

```diff
@@ -73,3 +72,36 @@ def __init__(self, bus, **kwargs):
             #                set page address     = 0xB0 - 0xBF (16 pages)
             SH1107_addressing=True,
         )
+        self.is_awake = True  # Display starts in active state (_INIT_SEQUENCE)
+
+    @property
+    def state(self):
+        """
+        The power state of the display. (read-only)
+
+        True if the display is active, False if in sleep mode.
+        """
+        return self.is_awake
```

Comment:
I think `self.is_awake` is enough because it can be read directly. `state` as a property name is vague so `is_awake` is better for that too. You can put this comment after the `self.is_awake =` line above to have it documented in the docs. The only reason to do it as a property would be to make it read-only (and then the internal variable should start with `_` so it's clear it's internal.)

Reply:
Thank you Scott, I appreciate your input (and patience).  This is my first PR and I don't have much experience with classes, so it is a learning experience!

Upon reading your request, I realize I poorly executed the implementation of what I intended which is a read-only property.  I don't think we want `is_awake` to be writable since the `wake()` and `sleep()` methods handle updating the state, and if the user changes `is_awake` to the incorrect state then the `wake()` and `sleep()` methods will not work correctly.  However, I didn't like using `state` as a property name since it is vague as you stated above.

In your opinion would the following be more appropriate / correct to achieve the goal?

    self._is_awake = True  # Display starts in active state (_INIT_SEQUENCE)

    @property
    def is_awake(self):
        """
        The power state of the display. (read-only)
        True if the display is active, False if in sleep mode.
        """
        return self._is_awake

And please feel free to give me a direct recommendation on how best to do this... you won't step on my toes or hurt my feelings!  As I said previously, I am learning and definitely want to do things the way you feel best!

Reply:
That's exactly what I was thinking! Nice work!

Reply:
Great, thanks for the feedback!  I will make the changes and push the update to the PR shortly.